schema/catvrs/catvrs-source.yaml

```diff
@@ -13,296 +13,254 @@ namespaces:
-      member CNVs.
+    inherits: Constraint
+    description:
+      a quantitative change in copy number in a system
     properties:
-      type:
-        extends: type
-        const: CategoricalCnv
-        default: CategoricalCnv
-        description: 'MUST be "CategoricalCnv"'
-      location:
-        type:
-          $refCurie: vrs:Location
-        description: >-
-          A `VRS Location <https://vrs.ga4gh.org/en/2.0/terms_and_model.html#location>`_
-          object that represents a sequence derived from that location, and is congruent with locations 
-          on alternate reference sequences.
-      locationMatchCharacteristic:
-        type: string
-        enum: ['exact', 'partial', 'subinterval', 'superinterval']
-        description: >-
-          The characteristics of a valid match between a contextual CNV location (the query) and the 
-          Categorical CNV location (the domain), when both query and domain are represented on the same 
-          reference sequence. An `exact` match requires the location of the query and domain to be identical. 
-          A `subinterval` match requires the query to be a subinterval of the domain. A `superinterval` match
-          requires the query to be a superinterval of the domain. A `partial` match requires at least 1 residue
-          of overlap between the query and domain.
-      copyChange:
-        type: string
-        enum: [ "efo:0030069", "efo:0020073", "efo:0030068", "efo:0030067", "efo:0030064", "efo:0030070", "efo:0030071", "efo:0030072" ]
-        description: >-
-          A representation of the change in copies of a sequence in a system. MUST be one of "efo:0030069" (complete 
-          genomic loss), "efo:0020073" (high-level loss), "efo:0030068" (low-level loss), "efo:0030067" (loss), 
-          "efo:0030064" (regional base ploidy), "efo:0030070" (gain), "efo:0030071" (low-level gain), "efo:0030072" 
-          (high-level gain).
       copies:
-        oneOf:
+        scale:
+          type: string
+          oneOf:
+            - absoluteCopyNumber
+            - foldChange
+            - log2Value
+        quantity:
+          oneOf:
           - type: integer
           - $refCurie: vrs:Range
-        description: >-
-          The integral number of copies of the subject in a system.
+        threshold:
+          type: string
+          description: >-
+            limit above/below which a completely different label may apply, eg CNGain in a gene is considered amplification above a certain threshold per BMH technical lab standards
+        copyChange:
+          $refCurie: gks.data-types:Coding
+          description: >-
+            MUST be a `Coding` representing one of "EFO:0030069" (complete genomic loss), "EFO:0020073" (high-level loss),"EFO:0030068" (low-level loss), "EFO:0030067" (loss), "EFO:0030064" (regional base ploidy),"EFO:0030070" (gain), "EFO:0030071" (low-level gain), "EFO:0030072" (high-level gain).
```

Comment:
Might be easy (or not) but consider constraining Coding.value to a set of the defined values above; will help with computable IDs if CatVRS is to support them.